docs/ROADMAP.md

```diff
@@ -1,4 +1,4 @@
-curl the next few years - perhaps
```

Comment:
I don't think `curl` should be capitalized, even when starting a sentence

Reply:
Nice to know, thanks and fixed.